cyclonedx_py/parser/poetry.py

```diff
@@ -52,12 +55,22 @@ def __init__(
         debug_message('processing poetry_lock')
         for package in poetry_lock['package']:
             debug_message('processing package: {!r}', package)
+
+            if package['category'] == "dev":
+                if "dev" in omit_category:
+                    debug_message("Ignoring development package!")
+                    continue
+
             purl = PackageURL(type='pypi', name=package['name'], version=package['version'])
             bom_ref = purl.to_string() if use_purl_bom_ref else None
             component = Component(
                 name=package['name'], bom_ref=bom_ref, version=package['version'],
                 purl=purl
             )
+            prop = Property(
+                name='cdx:poetry:package:group',
+                value=package['category'])
+            component.properties.add(prop)
```

Comment:
simplify to `component.properties.add(Property(...))

Reply:
Fixed